kcidb/orm/data.py

```diff
@@ -0,0 +1,302 @@
+from kcidb.orm.__init__ import Type, Relation,_DEFS, _CHECKOUT, _BUILD, _TEST, _INCIDENT, _ISSUE, _ISSUE_CULPRIT, _TEST_ENVIRONMENT
```

Comment:
This would create a circular dependency, and shouldn't happen.

We need to bring the definitions of these dependencies here as well (as I pointed out in the issue).

Additionally, please do not import from `__init__` directly. This creates a very tight "coupling" and makes the code more fragile.

Reply:
Okay, I will do that.